include/net/nrf_cloud.h

```diff
@@ -107,7 +107,9 @@ enum nrf_cloud_evt_type {
 	 *  be populated with a @ref nrf_cloud_disconnect_status value.
 	 */
 	NRF_CLOUD_EVT_TRANSPORT_DISCONNECTED,
-	/** A FOTA update has started. */
+	/** A FOTA update has started.
+	 * This event is only sent if CONFIG_NRF_CLOUD_FOTA_AUTO_START_JOB is enabled.
```

Comment:
```suggestion
	 * This event is only sent if @kconfig(CONFIG_NRF_CLOUD_FOTA_AUTO_START_JOB) is enabled.
```
This makes a link to the Kconfig option. Feel free to use the same for other instances, as well.

Reply:
nice, thanks, i will update all the CONFIG_ mentions in the file to use this.